packages/bot-web-ui/src/components/dashboard/dashboard.scss

```diff
@@ -1,6 +1,12 @@
 @import '../dashboard/dashboard-component/index.scss';
 @import '../dashboard/tutorial-tab/index.scss';
 
+.position-center {
+    display: flex;
+    justify-content: center;
```

Comment:
@rupato can we use here @mixin ?

Reply:
yes we can use mixin here. But decided to use extend so i can inherit it to the class below